samples/Rules.Framework.WebUI.Sample/ReadmeExample/BasicRulesEngineExample.cs

```diff
@@ -42,38 +42,31 @@ protected void AddRules(IEnumerable<RuleSpecificationBase<BasicContentType, Basi
                 .NewRule<BasicContentType, BasicConditionType>()
                 .WithName("Rule for January sample for premium clients.")
                 .WithContent(BasicContentType.FreeSample, "SmallPerfumeSample")
-                .WithCondition(c => c
-                    .AsValued(BasicConditionType.ClientType)
-                    .OfDataType<string>()
-                    .WithComparisonOperator(Operators.Equal)
-                    .SetOperand("Premium")
-                    .Build())
+                .WithValueCondition(BasicConditionType.ClientType, Operators.Equal, "Premium")
                 .WithDatesInterval(new DateTime(2023, 01, 01), new DateTime(2023, 02, 01))
                 .Build();
 
             var ruleForPremiumFreeSampleApril = RuleBuilder
                 .NewRule<BasicContentType, BasicConditionType>()
                 .WithName("Rule for April sample for premium clients.")
                 .WithContent(BasicContentType.FreeSample, "ShampooSample")
-                .WithCondition(c => c
-                    .AsValued(BasicConditionType.ClientType)
-                    .OfDataType<string>()
-                    .WithComparisonOperator(Operators.Equal)
-                    .SetOperand("Premium")
-                    .Build())
+                .WithValueCondition(BasicConditionType.ClientType, Operators.Equal, "Premium")
+                //.WithComposedCondition(LogicalOperators.And, x => x
```

Comment:
You may wanna remove the commented code

Reply:
Yes, this is a first commit to show the proposal and left here the code because it's the only sample now to see how it will be used.
The plan is to replace all other usages in the code by the streamlined methods, and maybe set the current methods as obsolete.